web/website/content/faq.md

```diff
@@ -6,11 +6,15 @@ Here are some of the most common questions we hear. Have something else you'd
 like to ask? Pop by our [Discord](https://discord.com/invite/eQcfaCmsNc) and ask
 away!
 
-{{< faq "Cool story Hansel, but what can I actually do with PRQL now?" >}}
+{{< faq "Cool story Hansel, but what can I actually do with PRQL _now_?" >}}
 
-We're still early, and the opportunities for using PRQL are focused on two
-integrations:
+PRQL builds powerful SQL queries today. In addition, we are focusing on several
+integrations. Here's how you can use PRQL:
```

Comment:
```suggestion
We're still early, and the opportunities for seamlessly writing PRQL are growing. We're focusing on several
integrations:
```

WDYT? I think it's important to stress that we're early and that more is coming....

Reply:
See above.